### PR TITLE
update translation

### PR DESCRIPTION
**Pull request by jjeroch**

### CODE REVIEW

**Comment by jjeroch** [comment]
@manojava-gk can you have a look how we get the links correctly displayed. I can not see it anywhere described. Currently it looks the following:
image

File: cx-portal/src/assets/locales/en/notification.json [hunk 1]
```diff
@@ -88,7 +88,8 @@
     "usecases": "Go to use cases",
```
Comment (manojava-gk) [inline cx-portal/src/assets/locales/en/notification.json:88]: in the code it is expecting usecase and not usecases.

It should contain all of these values
"link": {
    "appmarketplace": "Go to app marketplace",
    "technicalsetup": "Go to connector configuration",
    "home": "Go to home page",
    "app": "Go to app",
    "user": "Go to user",
    "usecase": "Go to use cases",
    "serviceprovider": "Go to service provider",
    "appmanagementboard": "Get there",
    "servicemanagementboard": "Get there",
    "appoverview": "Get there",
    "appsubscription": "Get theme",
    "adminboard": "Get there",
    "technicaluser": "Get there",
    "companyrolesserviceprovider": "Get there"
  },

File: cx-portal/src/assets/locales/en/notification.json [hunk 2]
```diff
@@ -88,7 +88,8 @@
...
     "serviceprovider": "Go to service provider",
     "appmanagementboard": "Get there",
-    "servicemanagementboard": "Get there"
+    "servicemanagementboard": "Get there",
+    "usermanagement": "...open the user management"
```
Comment (manojava-gk) [inline cx-portal/src/assets/locales/en/notification.json:92]: It should contain all of these values
"link": {
"appmarketplace": "Go to app marketplace",
"technicalsetup": "Go to connector configuration",
"home": "Go to home page",
"app": "Go to app",
"user": "Go to user",
"usecase": "Go to use cases",
"serviceprovider": "Go to service provider",
"appmanagementboard": "Get there",
"servicemanagementboard": "Get there",
"appoverview": "Get there",
"appsubscription": "Get theme",
"adminboard": "Get there",
"technicaluser": "Get there",
"companyrolesserviceprovider": "Get there"
},

**Comment by manojava-gk** [comment]
I thought of push this change to the PR directly. Unfortunately I'm not able to checkout to this branch.

capelit81991526:tx-portal-frontend mgkoush$ git checkout enabler/update-messages-(de-&-en)
-bash: syntax error near unexpected token `('

**Comment by jjeroch** [comment]
changes are done

**Review by manojava-gk — APPROVED** [review]
looks fine

**Pull request merged**